docs/documentation/models.md

```diff
@@ -8,43 +8,44 @@ npm i @axflow/models
 
 ## Features
 
-* First-class streaming support for both low-level byte streams and higher-level JavaScript object streams
```

Comment:
If we make these updates here, can we make the same ones in the models readme since these two files currently are the same (with the exception of the links are a little different)?

Reply:
done